test/distributed/_composable/test_composability/test_2d_composability.py

```diff
@@ -279,5 +380,419 @@ def parallelize(_model: Transformer, mesh: DeviceMesh, use_seq_parallel: bool):
         self.assertEqual(loss_no_cp2, loss_cp2)
 
 
+class Test2dParallelIntegration(DTensorTestBase):
```

Comment:
I think this should be named in a way that makes it clear that this is testing FSDP1.

Reply:
Renamed